mautic/mautic
# [UX] Change category Published label to “Available for use”

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Aiming for a more intuitive user experience, we identified an opportunity to enhance the label “Published” used for categories. A more precise label could better communicate the status of the category to users. The proposed solution in this pull request is to adopt the label “Available for use”, which clearly indicates that the category is ready for utilization.

Also considered:

Published/Unpublished: These labels are currently used. However, they may not be ideal for all Mautic features as these labels may not clearly convey whether the category is active and working.

Off/On: These labels are commonly used for binary states, such as a switch. However, they may not fully capture the status of a category in a marketing automation platform. The terms “Off” and “On” could be too vague and might not provide enough context about what they represent in terms of a category’s status.

In Preparation/Running: These labels could be suitable for processes or campaigns that have a clear start and end point. However, for a category, which is more of an organizational tool rather than a process, these terms might not be the most appropriate.

Draft/Enabled: While these labels could work for content items like blog posts or articles that have a drafting stage, they might not accurately represent the status of a category. A category doesn’t typically have a “draft” stage, so this could potentially lead to confusion.

The proposed label “Available for use” is more descriptive and directly indicates that the category is ready for utilization, which is why it was chosen over the other options.

Users managing categories stand to benefit from this change, as it provides a more straightforward understanding of the category status. Implementing this change could lead to a more seamless user experience when managing categories, potentially reducing any confusion about the status of the categories. This aligns with our ongoing efforts to make the platform more accessible and user-friendly.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Categories
3. Create a new category
4. Look at the new label on the last field

![image](https://github.com/mautic/mautic/assets/116097999/1fad2da5-67be-46fb-8f79-f84a919bbc39)
